README.md

```diff
@@ -47,6 +47,8 @@ pip install git+https://github.com/inseq-team/inseq.git
 
 Install extras for visualization in Jupyter Notebooks and 🤗 datasets attribution as `pip install inseq[notebook,datasets]`.
 
+Install [Jax](https://github.com/google/jax) (optional) with ```poetry install --with jax```.
```

Comment:
Provided that we do not use `jax` in the library but we only care for it to not crash our `jaxtyping`, we do not want to ask users to add it here, even if optionally. I think we should remove this.